fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/office/OfficeWorkbookPopulator.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.infrastructure.bulkimport.populator.office;
+
+import org.apache.fineract.infrastructure.bulkimport.populator.AbstractWorkbookPopulator;
+import org.apache.fineract.organisation.office.data.OfficeData;
+import org.apache.poi.hssf.usermodel.HSSFDataValidationHelper;
+import org.apache.poi.hssf.usermodel.HSSFSheet;
+import org.apache.poi.ss.SpreadsheetVersion;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.ss.util.CellRangeAddressList;
+
+import java.util.List;
+
+public class OfficeWorkbookPopulator extends AbstractWorkbookPopulator {
+    private  List<OfficeData> offices;
+
+    private static final int OFFICE_NAME_COL = 0;
+    private static final int PARENT_OFFICE_NAME_COL = 1;
+    private static final int PARENT_OFFICE_ID_COL=2;
+    private static final int OPENED_ON_COL = 3;
+    private static final int EXTERNAL_ID_COL = 4;
+    private static final int LOOKUP_OFFICE_COL=7;
+    private static final int LOOKUP_OFFICE_ID_COL=8;
+
+    public OfficeWorkbookPopulator(List<OfficeData> offices) {
+      this.offices=offices;
+    }
+
+    @Override
+    public void populate(Workbook workbook) {
+        Sheet officeSheet=workbook.createSheet("Offices");
+        setLayout(officeSheet);
+        setLookupTable(officeSheet);
+        setRules(officeSheet);
+        setDefaults(officeSheet);
+    }
+
+    private void setLookupTable(Sheet officeSheet) {
+        int rowIndex=1;
+        for (OfficeData office:offices) {
+            Row row=officeSheet.createRow(rowIndex);
+            writeString(LOOKUP_OFFICE_COL,row,office.name());
+            writeLong(LOOKUP_OFFICE_ID_COL,row,office.getId());
+            rowIndex++;
+        }
+    }
+
+    private void setLayout(Sheet worksheet){
+        Row rowHeader=worksheet.createRow(0);
+        worksheet.setColumnWidth(OFFICE_NAME_COL, 4000);
```

Comment:
don't use directly any value like 4000, define it as static variable and then use that . Do the same changes at other places also.